File: CS/TreeViewGettingStarted/ViewModel.cs
using DevExpress.Mvvm;
using System.Collections.ObjectModel;

namespace TreeViewGettingStarted {
    public class MainWindowViewModel : ViewModelBase {
        public MainWindowViewModel() {
            EmployeeDepartments = DepartmentData.GetDepartments();
        }
        public ObservableCollection<EmployeeDepartment> EmployeeDepartments { get; set; }
    }    
}

File: VB/TreeViewGettingStarted/ViewModel.vb
Imports DevExpress.Mvvm
Imports System.Collections.ObjectModel

Namespace TreeViewGettingStarted

    Public Class MainWindowViewModel
        Inherits ViewModelBase

        Public Sub New()
            EmployeeDepartments = GetDepartments()
        End Sub

        Public Property EmployeeDepartments As ObservableCollection(Of EmployeeDepartment)
    End Class
End Namespace
